build.gradle

```diff
@@ -88,9 +100,44 @@ task sources(type: Jar, dependsOn: classes) {
     from sourceSets.main.allSource
 }
 
+tasks.delombok.shouldRunAfter(sources)
+tasks.publish.dependsOn build
+tasks.build.shouldRunAfter(clean)
+tasks.javadoc.shouldRunAfter(clean)
```

Comment:
I'm a little hesitant to have javadoc and build run after clean...

@NotZachery thoughts?

Reply:
I don't see how this could become an issue so I'm fine by it.

Reply:
They are set to run after as should, not forced. This prevents (if you did, say, `./gradlew build clean javadoc` it from building then deleting the build.

Reply:
Ah okay - thanks!